### PR TITLE
Feature/check 2260 bulk remove tags from list view

### PR DESCRIPTION
**Pull request by melsawy**
## Description

This one to allow user to untag items from list view 

References: CHECK-2260

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can verify the changes. Please describe whether or not you implemented automated tests.

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

- File FFFF, line LL: This refactoring does this and this. Is it consistent with how it’s implemented elsewhere?
- Etc.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "id": "BulkActionsRemoveTag.success",
+    "description": "Success message for bulk untagging items",
+    "defaultMessage": "TODO: add success message."
```
Comment (amoedoamorim) [inline localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json:5]: Please review TODO

File: localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  {
+    "id": "tagMenu.search",
+    "defaultMessage": "Search…"
```
Comment (amoedoamorim) [inline localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json:9]: Missing description

File: localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  {
+    "id": "tagMenu.notFound",
+    "defaultMessage": "No tags found"
```
Comment (amoedoamorim) [inline localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json:13]: Missing description

File: localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "defaultMessage": "Tags will be removed from eligible items"
+  },
+  {
+    "id": "BulkActionsRemoveTag.dialogBody",
+    "description": "Body of dialog warning that bulk untagging items",
+    "defaultMessage": "The <b>{tags}</b> tag will be removed from selected items that have this tag. if a tag is not found in a selected item, the item will be unaffected.<br /><br /><b>This action cannot be undone.</b> Are you sure you want to continue?"
+  },
+  {
+    "id": "BulkActionsRemoveTag.proceedLabel",
+    "defaultMessage": "Remove tags"
```
Comment (amoedoamorim) [inline localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json:32]: Missing description

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 5]
```diff
@@ -0,0 +1,176 @@
+/* eslint-disable @calm/react-intl/missing-attribute */
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:1]: Please remove eslint-disable directive

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+  const handleRemoveTag = () => setShowDialog(true);
+
+  const handleOnCancel = () => {
+    setShowDialog(false);
+  };
+
+  return (
+    <React.Fragment>
+      <FormattedMessage id="tagMenu.search" defaultMessage="Search…">
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:94]: please use id="bulkActionsRemoveTag.search" and add a description

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+            cancelLabel={<FormattedMessage {...globalStrings.cancel} />}
+            inputPlaceholder={placeholder}
+            selected={[]}
+            onDismiss={onDismiss}
+            options={options}
+            onSubmit={handleRemoveTag}
+            onSelectChange={handleSelectChange}
+            notFoundLabel={
+              <FormattedMessage
+                id="tagMenu.notFound"
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:107]: please use id="bulkActionsRemoveTag.notFound" and add a description

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+                defaultMessage="No tags found"
+              />
+            }
+            submitLabel={
+              <FormattedMessage
+                id="bulkActionsTag.submitLabel"
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:113]: please use id="bulkActionsRemoveTag.submitLabel"

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+              />
+            }
+          />
+        )}
+      </FormattedMessage>
+      <ConfirmProceedDialog
+        open={showDialog}
+        title={
+          <FormattedMessage
+            id="BulkActionsRemoveTag.dialogTitle"
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:126]: please use id="bulkActionsRemoveTag.dialogTitle"

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+            defaultMessage="Tags will be removed from eligible items"
+            description="Title of dialog warning after bulk untagging items"
+          />
+        }
+        body={
+          <Typography variant="body1" component="p" paragraph>
+            <FormattedHTMLMessage
+              id="BulkActionsRemoveTag.dialogBody"
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:134]: please use id="bulkActionsRemoveTag.dialogBody"

File: src/app/components/media/BulkActionsRemoveTag.js [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+              }}
+              description="Body of dialog warning that bulk untagging items"
+            />
+          </Typography>
+        }
+        onProceed={handleSubmit}
+        onCancel={handleOnCancel}
+        proceedLabel={
+          <FormattedMessage
+            id="BulkActionsRemoveTag.proceedLabel"
```
Comment (amoedoamorim) [inline src/app/components/media/BulkActionsRemoveTag.js:147]: please use id="bulkActionsRemoveTag.proceedLabel" and add a description

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Looks great Sawy, but please review these minor issues flagged!

File: src/app/components/media/BulkActionsRemoveTag.js (hunk 5 above)
Comment (melsawy) [reply src/app/components/media/BulkActionsRemoveTag.js:1]: @alexmeedan removing this line shows `don't use spread operator  @calm/react-intl/missing-attribute` error

**Pull request merged**